lib/control/v1/conqueso.js

```diff
@@ -120,9 +120,19 @@ function Conqueso(app, storage) {
   }
 
   route.get((req, res) => {
+    const prop = req.params['0'].replace(new RegExp('/', 'g'), '');
```

Comment:
I think there's a better way to handle this. Instead of writing your own regexp, add handlers for both the URI generated by the init script and the single-property endpoint, then ignore any unnecessaery dynamic URI parameters:

```javascript
app.get('/v1/conqueso', all_properties)
app.get('/v1/conqueso/api/:role/properties', all_properties)
app.get('/v1/conqueso/api/:role/properties/:property', single_property)
```